nml/main.py

```diff
@@ -21,10 +21,7 @@
 try:
     from PIL import Image
 except ImportError:
-    try:
-        import Image
-    except ImportError:
-        pass
+    pass
```

Comment:
Why do these ImportError blocks remain? Is it actually possible to use NML without PIL? Why has the import in real_sprite.py not got an try/except block around it?

Reply:
It's probably possible to use NML without PIL, but it's not useful.

Reply:
See the commit message:

> Note that in some places, an ImportError for Pillow was and still is
ignored, since Pillow is not required for all codepaths. See commit
5612c5b where this was introduced.

And from that commit's message:

> This allows writing townname grfs without having PIL

Reply:
I'd be prepared to bet money that doesn't work (especially now that Pillow is listed as a requirement in setup.py), but sure, ok :)